requirements/requirements.txt

```diff
@@ -1,18 +1,15 @@
-Flask==0.10.1
+Flask==1.1.1
```

Comment:
Looks good, but can the requirements update be made into a separate commit via `git reset -p` or similar?